spec/receipt_spec.rb

```diff
@@ -87,4 +88,25 @@
                                                                cancellation_reason: nil }])
     end
   end
+
+  describe 'sandbox fallback' do
```

Comment:
Extra test to verify that, if Venice receives a sandbox receipt, then it will verify on development after getting a `21007` from production.